docs/data-collection/tear-down.md

```diff
@@ -1,94 +1,130 @@
-    - [ ] Exit and close the external door.
+- [ ] Lock the head coil back with its bottom part without plugging the connectors.
+- [ ] Put the head coil away with the other head-coils on the shelf next to the scanner.
 
-## AFTER SCAN, outside scanner room
+    !!! important "If the next user is already there or you know the current coil will be also used in the next session, plug it back into the bed socket."
 
-- [ ] Switch off the projector.
+- [ ] Return the bed to its *Home* position by pressing the :fontawesome-solid-house: button ([more info](scanning-notes.md#standard-extraction-of-the-participant)).
+- [ ] Take the ET arm outside to the Control Room and place it in a stable place.
+- [ ] Take the ECG electrodes, ECG cable, the RB, and the plexiglass panel outside to the control room.
+- [ ] Exit and close the external door.
+
+## Clearing up the Control Room
+
+### Finalize the *boxing* of the ET elements
+- [ ] Put the lens cover.
+- [ ] Unscrew the ET lens, while **ALWAYS** keeping one hand under the lens while screwing/unscrewing it and put it back into its pouch.
+    ![cover-mri-compatible-lens](../assets/images/cover-mri-compatible-lens.png "Cover MRI compatible lens")
+- [ ] Store the ET arm in its designated box.
+
+    !!! danger "REMEMBER | the infrared mirror is inside that box already: DO NOT crush the mirror."
+
+### Finalize the *boxing* of other physiological recording instruments
+
+- [ ] Store the ECG cable and the pouch with the ECG leads in its designated box / storage.
+- [ ] Carefully roll the fiber and power cable of the ET, and place them in the rolling table of the ET PC.
+- [ ] Carefully roll the GA tube and put it in its designated bag.
+- [ ] Carefully roll the RB tube and put it in its designated bag.
+- [ ] Disconnect the MMBT-S Interface, and the two corresponding cables (pink USB, 25-pin parallel), and insert them in their designated bags.
+- [ ] Remove the cable connecting the GA to the AMI100D module of the BIOPAC, roll it and put it in its designated bag.
+- [ ] Put the GA and the corresponding cables in the designated box.
+- [ ] Disconnect and bag the remaining cables connected to the BIOPAC:
+    - [ ] Ethernet to the *{{ secrets.hosts.acqknowledge | default("███") }}* computer.
+    - [ ] Power cord.
+- [ ] Retrieve and bag the *AcqKnowledge USB License Key* from the *{{ secrets.hosts.acqknowledge | default("███") }}* computer.
+- [ ] Store the *fMRI box*, BIOPAC and GA boxes back in room {{ secrets.rooms.et_camera| default("███") }}.
+
+### Finalizing with the ET's PC
 
 - [ ] **Retrieve ET recordings** (from {{ secrets.hosts.psychopy | default("███") }}):
-    - [ ] Insert a USB key into *{{ secrets.hosts.psychopy | default("███") }}* and save the experiment from AcqKnowledge.
+    - [ ] Insert a USB key into *{{ secrets.hosts.psychopy | default("███") }}*
     - [ ] Upload to a pre-designated drop-box (e.g., using Dropbox)
```

Comment:
I removed lines 113-115 since they were no longer relevant. I am considering moving the transfer to Dropbox under "data management".WDYT?

Reply:
I'd be more confident if we make sure the files are uploaded to Dropbox on the spot.

We should change the psychopy experiments to write directly on the shared folder instead of in the git repo.

Reply:
See TheAxonLab/HCPh-fMRI-tasks#24